src/site/controller.ts

```diff
@@ -11,9 +11,10 @@ import { verifySiteId } from './utils';
 const SiteRouter = Router();
 export default SiteRouter;
 
-SiteRouter.get(['/:siteId'], (res, req, next) =>
+SiteRouter.get(['/:siteId'], async (res, req, next) =>
```

Comment:
Does this need to be async if the function called here doesn't have an await?

Reply:
nope